js/src/lib/util.js

```diff
@@ -245,3 +261,16 @@ inlineRenderer.paragraph = function(text) {
 export const safeMarkdown = input => ({
   __html: xss(marked(unescape(input), { renderer: inlineRenderer })),
 });
+
+export const i18nReplaceVars = (message, vars) =>
```

Comment:
do you think you can reuse this in other places where we replace variables?

Reply:
Sure!

Reply:
Feel free to open another PR after this one reusing this function 🙌